### PR TITLE
Adds the National Ice Cream Day.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
This PR adds the National Ice Cream Day (yeah, it's one of those 'murrican holidays) to the game, bringing a few niche ice-cream-related effects to the game

During the Ice Cream Day:
- Crematoriums replaced with cre**a**matoriums.
- All cyborgs that don't already come with the treat dispenser are given (a slightly weaker) one stuck in ice cream mode.
- Robot customers order ice cream thrice as often.
- The ice cream vat starts with two times and a half more ingredients.
- Ice cream cones can hold up to four times as many scoops as normal.

## Why It's Good For The Game
I've got the idea after Burger closed his april fools creamatorium PR. Orange was right to say that it didn't fit well within aprils fools, but I digress...

We have just enough ice cream features to enstablish the holiday into the game without it passing as mere station prefixes fluff (also apparently it's a real, presidentally proclamated US holiday).

## Changelog

:cl:
add: Add the National Ice Cream Day holiday, which falls on the third Sunday of July.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
chrome_PzEiEVnVAO

**Comment by Boopideedoo** [comment]
This is a great Idea! I hope it gets merged for us to have national Ice cream Day!

File: code/modules/holiday/nth_week.dm [hunk 1]
```diff
@@ -88,6 +88,7 @@
 /datum/holiday/nth_week/ice_cream
 	name = ICE_CREAM_DAY
 	begin_week = 3
+	begin_month = JULY
```
Comment (Iamgoofball) [inline code/modules/holiday/nth_week.dm:91]: i disagree lets go back to every third sunday

Comment (Ghommie) [reply code/modules/holiday/nth_week.dm:91]: Only if oranges agrees.

**Comment by optimumtact** [comment]
>  Orange was right to say that it didn't fit well within aprils fools, but I digress...

I didn't say anything like that?

**Comment by Ghommie** [comment]
Oh, Then I either remembered it wrong or misread it.

**Comment by TheVekter** [comment]
Absolutely based, this has also reminded me that I wanted to do similar effects for Hot Dog Day but couldn't figure out how to make the game UI an eye-searing combination of red and yellow.

**Comment by Ghommie** [comment]
Learn color matrices. Also the Hot Dog Day already has something, thought it isn't much: It's a hot dog vending machine.

**Pull request merged**